_includes/navigation.html

```diff
@@ -1,6 +1,17 @@
 <!-- Navbar -->
 <nav class="navbar navbar-default navbar-expand-lg fixed-top" color-on-scroll="100">
     <div class="container">
+        <div class="navbar-translate">
```

Comment:
Could you please reformat this part? Seems like a div tag is missing.

If you are using VSCode, you can select this newly added part and right-click to find the formatting option

Reply:
You are telling to format the code right Shift+Alt+F right?

Reply:
@Piumal1999